CodeEditModules/Modules/WorkspaceClient/src/Model/FileItem.swift

```diff
@@ -257,7 +257,9 @@ public extension WorkspaceClient {
             var fileUrl = self.url
             while FileItem.fileManger.fileExists(atPath: fileUrl.path) {
                 let previousName = fileUrl.lastPathComponent
-                fileUrl = fileUrl.deletingLastPathComponent().appendingPathComponent("\(previousName) copy")
+                let filextension = fileUrl.pathExtension
+                let fileName = filextension.isEmpty ? previousName : previousName.replacingOccurrences(of: ".\(filextension)", with: "")
```

Comment:
There's a SwiftLint error on this line (`line_length`)

Reply:
Wait, really? It worked fine for me.. I'll check it again as soon as I get home; there wasn't any error whenever I built it.

Reply:
Do you have SwiftLint installed? [Code Style](https://github.com/CodeEditApp/CodeEdit/wiki/Code-Style#swiftlint)

Reply:
Oh, no wonder, I don't... The error seems to be: 
`Error: Line Length Violation: Line should be 120 characters or less: currently 136 characters (line_length)`
So do I just shorten the line somehow or...

Reply:
@lukepistrol ?